resources/js/views/DashboardView.vue

```diff
@@ -145,18 +149,49 @@ export default {
         console.log("error ", e)
       }
     },
+    async isGroupTooSmall(group, type){
+      let orgNr = group.description.split(":").at(-1)
+      let apiResult;
+      if(type == "school"){
+        let url = "/school/orgnr/" + orgNr;
+        apiResult = await api.get(url, {
+          params: { cookie: window.cookie }
+        });
+      }
+      else if(type == "kindergarten"){
+        let url = "/kindergarten/orgnr/" + orgNr;
+        apiResult = await api.get(url, {
+          params: { cookie: window.cookie }
+        });
+      }
+      if(apiResult.data.result.AnsatteTil <= 5){
+        return true;
+      }
+      return false;
+    },
     async getSurveyData() {
```

Comment:
Refactoring: The "getSurveyData" function is quite lengthy and could be refactored to improve readability and maintainability.